sql/seta_schema.sql

```diff
@@ -2,7 +2,7 @@ USE ouija;
 
 CREATE TABLE IF NOT EXISTS `seta` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `jobtype` varchar(64) NOT NULL,
+  `jobtype` varchar(1024) NOT NULL,
```

Comment:
It shows up one or two times when I insert data into seta table, I can't remember the special jobtype name now. But I could find it out later :)